documentation/docs/decentralized_identity.md

```diff
@@ -1,99 +1,103 @@
 
 ## Identity for People
 
-:::info New internet
+:::note 
 
 IOTA Identity builds a new internet, without usernames, passwords, endless repeated forums, or uncontrolled data harvesting.
 
 :::
 
-Information about one’s life is spread across many locations. Most people have numerous unorganized important documents at home, hundreds of online accounts, and many more online footprints. Through statistical predictive analysis, computer programs can harvest unverified online information sources and create a reasonably accurate profile about our lives. These profiles are accurate enough for targeted advertising and personalized content, but lack the proof and trust for them to be used in business. This results in an antiquated customer experience where we have to submit our age and address for every purchase we make and every account we create. It also inhibits our ability to do many online tasks like requesting and extending licenses or taking out a mortgage.
+Information about anyone's life is spread across many locations. Most people have numerous unorganized important documents at home, hundreds of online accounts, and many more online footprints. Through statistical predictive analysis, computer programs can harvest unverified online information sources and create a reasonably accurate profile about our lives. These profiles are accurate enough for targeted advertising and personalized content but lack the proof and trust for them to be used in business. This results in an antiquated customer experience where we have to submit our age and address for every purchase we make and every account we create. It also inhibits our ability to do many online tasks like requesting and extending licenses or taking out a mortgage.
 
-Self-Sovereign Identity is about returning autonomy and privacy to the individual, while also improving our online experience. Some movements focus on data privacy, preventing companies from using our information all together, but the IOTA Identity allows controlled information flow. The user can create a single online profile, containing all our personal information. They can decide who they share what information with, and a Verifier is able to verify the information to be correct, making the data trustworthy. This moves their online profile from a statistical estimation by corporate entities to an accurate and verifiable profile under their own control.
+Self-Sovereign Identity is altogether, but the IOTA Identity allows for a controlled flow of information. The user can create a single online profile containing all our personal information. They can decide who they share what information with, and a Verifier verifies if the information is correct, making the data trustworthy. This moves their online profile from a statistical estimation by corporate entities to an accurate and verifiable profile under their own control.
```

Comment:
I like the idea, but I think that "Self-Sovereign Identity is altogether" is a fair bit confusing. "SSI contains all information, but you control which part of it you want to reveal with IOTA DID" or along the lines?

BTW I really like that Jelle is a loudmouth. It does help to understand why DIDs are so cool and important and all.

Reply:
> Verifier verifies

Disliker dislikes :-1: